lib/pf/UnifiedApi/Controller/Config/FirewallSsoes.pm

```diff
@@ -0,0 +1,94 @@
+package pf::UnifiedApi::Controller::Config::FirewallSsoes;
```

Comment:
I think this one should be a special case and be called firewalls_sso

Reply:
How about sso_firewalls

Reply:
Or maybe we could just strip the SSO and just call it firewalls

Reply:
If we use _policy / _policies for traffic shaping, how about firewall_policy / firewall_policies ?